app/controllers/tourist_areas_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TouristAreasController < ApplicationController
+  def index
+  	@tourist_areas = TouristArea.all
```

Comment:
không lấy toàn bộ record thế này. phân trang để đảm bảo hiệu năng (will_paginate, kaminari, pagy tuỳ chọn 1 trong 3)